api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java

```diff
@@ -487,28 +488,20 @@ public Relationship saveRelationship(Relationship relationship) throws APIExcept
 	 *      java.lang.String)
 	 */
 	@Override
-	public Relationship voidRelationship(Relationship relationship, String voidReason) throws APIException {
-		if (relationship.getVoided()) {
-			return relationship;
-		}
-		
-		relationship.setVoided(true);
-		if (relationship.getVoidedBy() == null) {
-			relationship.setVoidedBy(Context.getAuthenticatedUser());
-		}
-		if (voidReason != null) {
-			relationship.setVoidReason(voidReason);
+	public Relationship voidRelationship(@NonNull final Relationship relationship, String voidReason) throws APIException {
+		if (relationship ==null) {
```

Comment:
we can have some space when using null for example  if (relationship == null) {

Reply:
thank you for this correction @HerbertYiga